<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClusteringApp</class>
 <widget class="QMainWindow" name="ClusteringApp">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clustering App</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabAll">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>781</width>
      <height>631</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_input">
     <attribute name="title">
      <string>Dataset</string>
     </attribute>
     <widget class="QTextEdit" name="dirtext">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>441</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Đường dẫn:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="browserDir">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>20</y>
        <width>93</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Browser...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loadFile">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>20</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>131</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Số lượng mẫu:</string>
      </property>
     </widget>
     <widget class="QLabel" name="n_Sample">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>100</y>
        <width>101</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>141</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Các thuộc tính:</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listViewField">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>191</y>
        <width>751</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="n_Field">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>140</y>
        <width>111</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_dataset_base">
     <attribute name="title">
      <string>Tập giống</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>521</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Tập giống được phân loại theo cột nào? </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>221</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Vui lòng thêm tập giống:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="getBaseInDataset">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>40</y>
        <width>181</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Lấy dữ liệu từ dataset</string>
      </property>
     </widget>
     <widget class="QPushButton" name="getBaseInFile">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>40</y>
        <width>101</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Thêm từ file</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbb_FieldBase">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>120</y>
        <width>401</width>
        <height>40</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="fieldBaseLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>741</width>
        <height>80</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="n_clusterBaseLabel">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>330</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="fieldBaseLabel_">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>201</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>13</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Phân loại theo cột: </string>
      </property>
     </widget>
     <widget class="QLabel" name="n_clusterBaseLabel_">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Số cụm của tập giống: </string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidgetFeildBase">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>751</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabRun">
     <attribute name="title">
      <string>Khởi chạy</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>171</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Số cụm tối đa:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="kMax">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>104</width>
        <height>40</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>421</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string> Các trường dùng để phân loại:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_Run">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>40</y>
        <width>151</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Chạy</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidgetField4Run">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>741</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>550</y>
        <width>751</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabResult">
     <attribute name="title">
      <string>Kết quả</string>
     </attribute>
     <widget class="QWidget" name="widgetPieChart" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>751</width>
        <height>441</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_saveFile">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>530</y>
        <width>111</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Xuất ra tệp</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_detail">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>530</y>
        <width>93</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Chi tiết</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabPredict">
     <attribute name="title">
      <string>Dự đoán</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidgetPredict">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>740</width>
        <height>521</height>
       </rect>
      </property>
      <property name="tabKeyNavigation">
       <bool>true</bool>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Feilds</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Options</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="btnResetPredict">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>550</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Đặt lại</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
